README.md

```diff
@@ -1,2 +1,26 @@
 # localizable-manifests
 Definitions and best practices related to the specification of manifest files and similar document formats on the Web.
+
+### Documents
+- [Editor's copy](https://w3c.github.io/localizable-manifests/)
+- [WD](https://www.w3.org/TR/localizable-manifests/)
+
+### Feedback
+Please use the [GitHub issue list](https://github.com/w3c/localizable-manifests/issues) to send feedback about this document.
+
+To make it easier to track comments, please raise separate issues or emails for each comment, and point to the section you are commenting on  using a URL for the dated version of the document.
+
+### Following
+To follow the work, you can 'Watch' this repository using the control above, or subscribe to the [www-international](https://lists.w3.org/Archives/Public/www-international/) mailing list, which is notified once a day about changes to the repo. Please use github issues rather than the mailing list to send feedback.
+
+### Contributing
+
+All contributors should read and agree with [CONTRIBUTING.md](https://github.com/w3c/localizable-manifests/blob/gh-pages/CONTRIBUTING.md).
+
+Editors should be familiar with and use the following:
+
+- [Github guidelines for working with i18n documents](https://w3c.github.io/i18n-activity/guidelines/github)
```

Comment:
This and following links with i18n-activity in the path should be changed to https://www.w3.org/International/i18n-activity/....